sql/moz-fx-data-shared-prod/apple_ads_external/ad_group_report_v1/query.sql

```diff
@@ -11,8 +11,8 @@ SELECT
   start_at,
   end_at,
   taps,
-  new_downloads,
-  redownloads,
+  tap_new_downloads AS new_downloads,
+  tap_redownloads AS redownloads,
   total_downloads,
```

Comment:
[This comment in the dbt model code](https://github.com/fivetran/dbt_apple_search_ads/blob/ba2f18e161fb904c192a7268ed69647999032920/models/apple_search_ads__ad_group_report.sql#L49-L50) says `total_downloads` will be deprecated and `tap_total_downloads` should be used instead.
```suggestion
  tap_total_downloads AS total_downloads,
```
(and the same goes for the other tables)